website/docs/reference/widgets/table/inline-editing.md

```diff
@@ -235,13 +235,117 @@ This property contains the details of the row that triggered the action (`onSubm
+
+### Properties
+
+The following properties are related to adding new rows:
+
+| Property              | Type        | Definition               | Code Snippet        |
+| --------------------- | ----------- | ------------------------ | ------------------- |
+| **Allow adding a row** | Widget | Toggles a button in the table which allows users to submit new rows of data. Only columns marked as **Editable** can accept user input. Use the **onSave** event to update the source of the table's data and reflect the user's changes. |
+| **Default Values** | Widget | The values to automatically populate the new row with when a user begins creating a new row. Expects an object with the same keys as the columns in the existing table data. |
+| **isAddRowInProgress** | Binding | Indicates whether a new row is currently being added by the user. | `Table1.isAddRowInProgress` |
+| **newRow** | Binding | When writing code for validation properties, this variable contains a reference to the new row object added by the user. | `Table1.newRow` |
+| **isNewRow** | Validation | When writing code for validation properties, this variable indicates whether the cells being validated are part of a new or an existing row. | `isNewRow` |
+
+The subheadings below describe these properties in more detail:
+
+#### Allow adding a row
+
+Turning this on allows users to click a button on the table to start a new row in the table. Users can only input data for columns that are marked as **Editable** in the table properties.
+
+Use the **onSave** event to update the source of the table's data and reflect the user's changes. For example, the **onSave** event might be set to trigger a query that sends the data from a new table row to the database that supplies the table with its values.
+
+#### Default Values
+
+<VideoEmbed host="youtube" videoId="3KozovP7LNk" title="Table | Add new rows | Default values" caption="Set default values for new Table rows"/>
+
+The values to automatically populate the new row with when a user begins creating a new row. Expects an object with the same keys as the columns in the existing table data.
+
+#### isAddRowInProgress
+
+While the "Add new row" mode is enabled in the Table widget, `isAddRowInProgress` is `true`. You can use this for a variety of reasons, such as providing visual feedback in the UI for the user or for enforcing specific validation rules for user input in new rows (See [isNewRow](#isnewrow)).
+
+#### newRow
+
+When a new row is being added to the Table, this variable contains a reference to that new row object being added. When there is not a new row in progress, `newRow` is `undefined`.
+
+This is useful for accessing the user input to send it via query once the user is ready to submit the new data.
+
+#### isNewRow
+
+`isNewRow` can be used only within the **Valid** property. With it, you may set specific validation rules for new table rows that does not neccesarily apply to editing existing cells.
```

Comment:
I think this is incorrect. It can be used in all validation related properties of the column. i.e Regex, Valid, Error Message, Required. Can we confirm this with @sbalaji1192 ?

Reply:
Yes. All validation properties have access to the isNewRow variable.